.gitattributes

```diff
@@ -0,0 +1,17 @@
+# This .gitattributes file will cause all text files EXCEPT for
+# git's .gitattributes and .gitignore files to be encoded as EBCDIC.
+# Selected binary files will not be translated at all.
+# The default for text files
+* git-encoding=iso8859-1 working-tree-encoding=iso8859-1
```

Comment:
I am not that familiar with `.gitattributes` but the comments are not correct.  The comments about EBCDIC are clearly only true on zOS.  Is this going to be an issue on platforms other than zOS?

Reply:
@charliegracie No, the extra git-encoding in .gitattributes is only used when using the version of git that is available for z/OS from Rocket Software. All other platforms will ignore these options.

Reply:
Have modified the comments in this file to make it clear that this only effects z/OS platform.